app/helpers/javascript_helper.rb

```diff
@@ -0,0 +1,5 @@
+module JavascriptHelper
+  def initialise_script(constructor_or_singleton_name, params = {})
+    content_for :javascript_initialisers, raw("GOVUK.init(#{constructor_or_singleton_name}, #{params.to_json});\n");
```

Comment:
This doesnt look right, there isnt a function to called init on here.

Doing eval of javascript like this is probably not a good idea either.